workspace_integration_test.go

```diff
@@ -1173,6 +1173,70 @@ func TestWorkspacesReadByID(t *testing.T) {
 	})
 }
 
+func TestWorkspacesAddTagBindings(t *testing.T) {
+	skipUnlessBeta(t)
+
+	client := testClient(t)
+	ctx := context.Background()
+
+	wTest, wCleanup := createWorkspace(t, client, nil)
+	t.Cleanup(wCleanup)
+
+	t.Run("when adding tag bindings to a workspace", func(t *testing.T) {
+		tagBindings := []*TagBinding{
+			{Key: "foo", Value: "bar"},
+			{Key: "baz", Value: "qux"},
+		}
+
+		bindings, err := client.Workspaces.AddTagBindings(ctx, wTest.ID, WorkspaceAddTagBindingsOptions{
+			TagBindings: tagBindings,
+		})
+		require.NoError(t, err)
+
+		assert.Len(t, bindings, 2)
```

Comment:
:nail_care: 
```suggestion
		require.Len(t, bindings, 2)
```
to prevent a panic downstream